tool/tctl/common/proxy_command.go

```diff
@@ -0,0 +1,66 @@
+package common
+
+import (
+	"context"
+	"github.com/gravitational/kingpin"
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/service"
+	"github.com/gravitational/trace"
+	"io"
+	"os"
+)
+
+// ProxyCommand returns information about connected proxies
+type ProxyCommand struct {
+	config *service.Config
+	lsCmd  *kingpin.CmdClause
+
+	format string
+
+	stdout io.Writer
+}
+
+// Initialize creates the proxy command and subcommands
+func (p *ProxyCommand) Initialize(app *kingpin.Application, config *service.Config) {
+	p.config = config
+
+	auth := app.Command("proxy", "Operations with information for cluster proxies").Hidden()
+	p.lsCmd = auth.Command("ls", "List connected auth servers")
+	p.lsCmd.Flag("format", "Output format: 'yaml', 'json' or 'text'").Default(teleport.YAML).StringVar(&p.format)
```

Comment:
the usual default is text, is there a reason to go with yaml here?

Reply:
For this the GitHub issue indicated that the output should look like it already does for the other `ResourceCommand` `get` commands - which are `YAML` already: 

```
	rc.getCmd.Flag("format", "Output format: 'yaml', 'json' or 'text'").Default(teleport.YAML).StringVar(&rc.format)
```

I could change this one or both but the latter seems like a backwards incompatible change and the former would introduce an inconsistency between similar outputs. Thoughts?